xlive/Util/log.cpp

```diff
@@ -19,6 +19,9 @@ h2log::h2log(const std::string& name)
 {
 	sname = name;
 	wname = string_convert.from_bytes(name);
+
+	// should be fine to do this for every logger, better here once than someplace else
```

Comment:
second part of the doc seems unnecessary

if you actually have a performance concern calling flush everytime a log is constructed then I'd question this entire line. It really depends how often we are constructing log objects, if its not often, the doc isn't necessary.